keyboards/telophase/rules.mk

```diff
@@ -1,15 +1,64 @@
+
+OPT_DEFS += -DTELOPHASE_PROMICRO
+OPT_DEFS += -DCATERINA_BOOTLOADER
+INTERPHASE_UPLOAD_COMMAND = while [ ! -r $(USB) ]; do sleep 1; done; \
+                         avrdude -p $(MCU) -c avr109 -U flash:w:$(TARGET).hex -P $(USB)
```

Comment:
These aren't needed or used. QMK provides a way to flash keyboards intelligently based off the bootloader defined, with `make keyboard:keymap:flash`.